plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -2407,6 +2407,55 @@ public void testDynamicFilteringCaseInsensitiveDomainCompaction()
         }
     }
 
+    @Test
+    public void testUpdateProcedure()
+    {
+        String tableName = "test_update" + randomNameSuffix();
```

Comment:
Should it be skipped based on behaviour?
skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE) && ....);

Reply:
This test should be executed regardless the connector's CREATE TABLE support because it's unrelated to this procedure.